### PR TITLE
Update cmd/config, kyaml and cli-utils dependencies

### PR DESCRIPTION
**Pull request by mortent**
Update the dependencies

### CODE REVIEW

File: commands/livecmd.go [hunk 1]
```diff
@@ -64,7 +64,7 @@ func GetLiveCommand(name string, f util.Factory) *cobra.Command {
 	applyCmd.Long = livedocs.ApplyShort + "\n" + livedocs.ApplyLong
 	applyCmd.Example = livedocs.ApplyExamples
 
-	previewCmd := preview.GetPreviewRunner(f, ioStreams).Command
+	previewCmd := preview.PreviewCommand(f, ioStreams)
```
Comment (seans3) [inline commands/livecmd.go:67]: Let's keep the `Runner`. We will need it to inject the `ResourceGroup` versions of structures with the `ResourceGroupProvider`.

File: commands/livecmd.go [hunk 2]
```diff
@@ -74,7 +74,7 @@ func GetLiveCommand(name string, f util.Factory) *cobra.Command {
 	diffCmd.Long = livedocs.DiffShort + "\n" + livedocs.DiffLong
 	diffCmd.Example = livedocs.DiffExamples
 
-	destroyCmd := destroy.GetDestroyRunner(f, ioStreams).Command
+	destroyCmd := destroy.DestroyCommand(f, ioStreams)
```
Comment (seans3) [inline commands/livecmd.go:77]: Same

**Review by seans3 — CHANGES_REQUESTED** [review]

File: commands/livecmd.go (hunk 1 above)
Comment (mortent) [reply commands/livecmd.go:67]: But we can just update that when we have the provider for ResourceGroup ready. This already is using the default provider.

**Review by seans3 — APPROVED** [review]
/lgtm

**Pull request merged**